src/components/Class/ClassCard/index.js

```diff
@@ -83,20 +78,10 @@ function ClassCard({ item, index, handleDropOutData, editClass }) {
         accept: "application/json",
         Authorization: user.data.token,
       },
-    }).then(
-      () => {
-        notify();
-        dispatch(classActions.deleteClass(data, id));
-      },
-      (error) => {
-        toast.error(
-          `Something went wrong with error status: ${error.response.status} ${error.response.data.message}`,
-          {
-            position: toast.POSITION.BOTTOM_RIGHT,
-          }
-        );
-      }
-    );
+    }).then(() => {
+      notify();
+      dispatch(classActions.deleteClass(data, id));
```

Comment:
we should not pass data as part of the action, it's already there in the reducer.

Reply:
https://github.com/navgurukul/bhanwari-devi/pull/85/commits/9feab343ff61492a0305dafdf53c22262ccd9e11